packages/firestore/test/integration/api/transactions.test.ts

```diff
@@ -375,15 +375,16 @@ apiDescribe('Database transactions', persistence => {
     return integrationHelpers.withTestDb(persistence, db => {
       const doc1 = db.collection('counters').doc();
       const doc2 = db.collection('counters').doc();
-      let tries = 0;
+      // let tries = 0;
```

Comment:
rm?

Reply:
This code is used in the commented out part of this tests that verifies the retry behavior. Removing it means that we cannot easily comment the block back in once b/34879758 gets addressed.

Reply:
In that case, I'd recommend adding a TODO(b/34879758) or similar stating this. Otherwise, I'll quickly forget this conversation and remove that code next time I touch this file.

Reply:
It's part of a TODO that is just below this block.

Reply:
(after clicking expand a few times...)

Oh, I see. Given that I missed it once, there's a good chance without a TODO I'd miss it again. However, in order to actually change it, I'd have to load it up in my editor rather than the github diff viewer, so maybe I'd see it in that situation.

(No action required.)